README.md

```diff
@@ -5,6 +5,7 @@ This repo integrates slamtec rplidar LIDARS within Viam's [RDK](https://github.c
 It has been tested on the following rplidars:
 * [RPLIDAR A1](https://www.slamtec.com/en/Lidar/A1)
 * [RPLIDAR A3](https://www.slamtec.com/en/Lidar/A3)
+* [RPLIDAR S1](http://bucket.download.slamtec.com/f19ea8efcc2bb55dbfd5839f1d307e34aa4a6ca0/LD601_SLAMTEC_rplidar_datasheet_S1_v1.4_en.pdf)
```

Comment:
[note] https://www.slamtec.com/en/Lidar/S1 is a deadlink as SLAMTEC does not support S1